src/updater.cpp

```diff
@@ -266,6 +266,7 @@ void Updater::setupOnLinux()
 void Updater::untarOnLinux(const QString& fileName)
 {
     Q_ASSERT_X(QThread::currentThread() != QCoreApplication::instance()->thread(), "untarOnLinux", "method should not be called in the main GUI thread to avoid a degradation in UX");
+    qWarning() << __func__ << "started";
```

Comment:
Surely this (and all the others) are `qDebug()`s (or a `qInfo()`s though we haven't been using that one) - this is not something that is indicating a fault in any way!

Reply:
Agreed, but I already started out with a qWarning() so was not sure if mixing the two would have effects for how it's printed in the end.

Reply:
Depends on whether one has a certain option set and one is running in the debugger...
![image](https://user-images.githubusercontent.com/6163092/114247593-617db980-998d-11eb-9b18-8f898e976f76.png)